### ISSUE TITLE
Error: Cannot find module 'webpack-dev-server/lib/util/createDomain'

### ISSUE THREAD

**Issue by nils4iviews** [webpack-contrib/grunt-webpack#167]
In the upgrade of `webpack-dev-server@3.1.6` to `webpack-dev-server@3.1.7`, the file 'lib/utils/createDomain.js` has been renamed to `lib/utils/createDomain.js` (util -> utils). 

The following line in `grunt-webpack` is using this file and is now failing:
https://github.com/webpack-contrib/grunt-webpack/blob/1162becd006dbc98a32e93b281e58fd42d205659/tasks/webpack-dev-server.js#L48

Appearently, the webpack-dev-server project team does not consider this file to be part of the interface, otherwise this should have been a major version bump.

**Comment by danez** [comment]
Fixed with #166

**Comment by realdennis** [comment]
```
Loading "webpack-dev-server.js" tasks...ERROR
>> Error: Cannot find module 'webpack-dev-server/lib/util/createDomain'
```

Looks like `createDomain` is not in the `webpack-dev-server` anymore
(the change commit in webpack-dev-server https://github.com/webpack/webpack-dev-server/commit/2d68701d642f848577c2c0bb73701e3c6727da1a)

See there's no util in `webpack-dev-server/lib` folder https://github.com/webpack/webpack-dev-server/tree/master/lib

better not use the dependency's utility from module
